controller/lib/core/blower_pid.cpp

```diff
@@ -42,24 +38,19 @@ static constexpr float Kd = Ku * Tu.seconds() / 15;
 
 // DIRECT means that increases in the output should result in increases in the
 // input.  DIRECT as opposed to REVERSE.
-static PID myPID(&Input, &Output, &Setpoint, Kp, Ki, Kd, DIRECT);
+static PID myPID(Kp, Ki, Kd, ProportionalTerm::ON_ERROR,
+                 DifferentialTerm::ON_MEASUREMENT, ControlDirection::DIRECT);
 
 // TODO: VOLUME_INTEGRAL_INTERVAL was not chosen carefully.
 static constexpr Duration VOLUME_INTEGRAL_INTERVAL = milliseconds(5);
 static Time last_flow_measurement_time = millisSinceStartup(0);
 static float last_flow_ml_per_min = NAN;
 
 void blower_pid_init() {
-  Setpoint = 0;
-  Input = 0;
-  Output = 0;
   myPID.SetSampleTime(PID_SAMPLE_TIME);
 
   // Our output is an 8-bit PWM.
   myPID.SetOutputLimits(0, 255);
```

Comment:
Should these be values set on construction?

Reply:
Done, removed the setters.